package.json

```diff
@@ -55,7 +55,7 @@
     "lint:eslint": "eslint src",
     "lint:stylelint": "stylelint \"./src/**/*.{ts,tsx}\"",
     "prettier": "prettier --write \"src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
-    "prettier:check": "prettier-check \"src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
+    "prettier:check": "prettier --check \"src/**/*.{js,jsx,ts,tsx,json,css,scss,md}\"",
```

Comment:
This should probably be changed for the `pre-commit` hook as well

Reply:
Good catch. Fixed now.